client/src/components/NewColumnDialogBox/NewColumnDialogBox.tsx

```diff
@@ -0,0 +1,104 @@
+import { FC } from 'react';
+import Button from '@material-ui/core/Button';
+import Dialog from '@material-ui/core/Dialog';
+import DialogContent from '@material-ui/core/DialogContent';
+import DialogTitle from '@material-ui/core/DialogTitle';
+import * as yup from 'yup';
+import { Formik, Form } from 'formik';
+import TextField from '@material-ui/core/TextField';
+import { useBoard } from '../../context/useBoardContext';
+import { useBackdrop } from '../../context/useBackDropContext';
+import IconButton from '@material-ui/core/IconButton';
+import CloseIcon from '@material-ui/icons/Close';
+import useStyles from './useStyles';
+import Typography from '@material-ui/core/Typography';
+
+interface ColumnDialogFormProps {
+  open: boolean;
+  handleClose: () => void;
+  details: any;
+}
+
+const NewColumnDialogBox: FC<ColumnDialogFormProps> = ({ open, handleClose, details }): JSX.Element => {
+  const { updateBoard } = useBoard();
+  const { backdropOpen, openBackdrop, closeBackdrop } = useBackdrop();
+  const classes = useStyles();
+  const doHandleClose = () => {
+    details.draggedCardColumn.cards.splice(details.draggedCardIndex, 0, details.draggedCard);
+    handleClose();
+  };
+
+  return (
+    <div>
+      <Dialog open={open} onClose={doHandleClose} aria-labelledby="form-dialog-title">
+        <DialogTitle disableTypography className={classes.title}>
+          <Typography variant="h6" className={classes.text}>
+            Create a new column
+          </Typography>
+          {open ? (
+            <IconButton aria-label="close" className={classes.closeButton} onClick={doHandleClose}>
+              <CloseIcon />
+            </IconButton>
+          ) : null}
+        </DialogTitle>
+        <DialogContent>
+          <Formik
+            initialValues={{ title: '' }}
+            validationSchema={yup.object({
+              title: yup.string().required('Title is required'),
+            })}
+            onSubmit={(values, { setSubmitting }) => {
+              setSubmitting(false);
+              openBackdrop();
+              setTimeout(() => {
+                closeBackdrop();
+              }, 1200);
+              console.log(backdropOpen);
```

Comment:
You can remove the console log

Reply:
Thank you very much @ajaytomgeorge I just removed the console log